src/js/contentscripts/socialwidgets.js

```diff
@@ -330,7 +330,7 @@ function replaceIndividualButton(tracker) {
 *                            replaced
 */
 function getTrackerData(callback) {
-  chrome.runtime.sendMessage({checkReplaceButton:document.location}, function(response) {
+  chrome.runtime.sendMessage({checkReplaceButton:document.location.hostname}, function(response) {
```

Comment:
Cool. Bugzilla is kind of a pain. For 
"product" you'll probably want "Toolkit" and for "component" whichever "WebExtensions" component seems to fit best.